Source/Urho3D/Graphics/CubemapCapture.cpp

```diff
@@ -0,0 +1,370 @@
+//
+// Copyright (c) 2017-2020 the rbfx project.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+#include "../Graphics/Camera.h"
+#include "../Core/Context.h"
+#include "../Graphics/CubemapCapture.h"
+#include "../Graphics/Graphics.h"
+#include "../Graphics/GraphicsEvents.h"
+#include "../IO/Log.h"
+#include "../Scene/Node.h"
+#include "../Graphics/RenderPath.h"
+#include "../Graphics/RenderSurface.h"
+#include "../Graphics/Renderer.h"
+#include "../Resource/ResourceCache.h"
+#include "../Scene/Scene.h"
+#include "../Graphics/TextureCube.h"
+#include "../Graphics/View.h"
+#include "../Graphics/Zone.h"
+
+#ifdef URHO3D_COMPUTE
+    #include "../Graphics/ComputeDevice.h"
+#endif
+
+#include "../DebugNew.h"
+
+#pragma optimize("", off)
```

Comment:
Why needed?

Reply:
The optimize-off is just a straggler that didn't get cleaned out after debugging, it has no legitimate reason to be there.